tck/src/test/java/org/reactivestreams/tck/PublisherVerificationTest.java

```diff
@@ -264,7 +264,7 @@ public void required_spec109_subscribeThrowNPEOnNullSubscriber_shouldFailIfDoesn
   }
 
   @Test
-  public void optional_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe_actuallyPass() throws Throwable {
+  public void required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe_actuallyPass() throws Throwable {
     customPublisherVerification(SKIP, new Publisher<Integer>() {
```

Comment:
That's OK, because this test is for the error state publisher. `customPublisherVerification` first the normal publisher, then the error state publisher - we're testing the error state publisher here, thus only the 2nd param.

// In the test we have `whenHasErrorPublisherTest(...)`